graphql/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Build Stage
+# =============================================================================
+FROM python:3.9-slim as builder
+
+WORKDIR /app
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# Install python app requirements
+COPY requirements.txt .
+RUN pip wheel --no-cache-dir --wheel-dir /app/wheels -r requirements.txt
+
+
+# Final Stage
+# =============================================================================
+FROM python:3.9-slim
+
+# Set metadata information
+LABEL application.name="ORCA-GraphQL" \
+    application.description="GraphQL server for ORCA application." \
+    application.team="ORCA" \
+    application.version="v0.0.16"
+
+# CREATE Non-Privileged User
+RUN mkdir -p /app \
```

Comment:
We could simplify this string of commands to something like this if we want.
```
RUN groupadd -r appuser \
    && useradd -r -s /bin/false -m -d /app -g appuser appuser
```

Reply:
Changed